### DISCUSSION TITLE
Question: How to read the best stories on a specific date? (/front?day=<date>)

### DISCUSSION THREAD

**Discussion by abceleung** [SimonHalvdansson/Harmonic-HN#145] [Q&A]
I like to read Hacker News like this:
1. Starting from [https://news.ycombinator.com/front](https://news.ycombinator.com/front), this shows me the best stories on yesterday (UTC)
2. Then I click the "go back a day" button and read the best stories on day before yesterday
3. Repeat until I finished reading

Can I replicate this flow in the Harmonic app? No existing filtering options in the app can give me the same result as my flow above.
Thanks!

**Comment by SimonHalvdansson** [comment]
Added to https://github.com/SimonHalvdansson/Harmonic-HN/issues/313, very reasonable request, thanks!
